.eslintrc.json

```diff
@@ -6,7 +6,7 @@
       "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
       "rules": {
         "@nrwl/nx/enforce-module-boundaries": [
-          "error",
+          "warn",
```

Comment:
I'm unsure why this was erroring , its a weird error that was picking up @angular/router as our own module. so i moved it to warning